test/pg_regress.sh

```diff
@@ -108,8 +108,14 @@ elif [[ -z ${TESTS} ]] && ( [[ -n ${SKIPS} ]] || [[ -n ${IGNORES} ]] ); then
     for test_pattern in ${SKIPS}; do
       for test_name in ${ALL_TESTS}; do
         if [[ $test_name == $test_pattern ]]; then
-          sed -i ${TEMP_SCHEDULE} -e "s!^test:\s*${test_name}\s*\$!!"
-          sed -i ${TEMP_SCHEDULE} -e "s!\b${test_name}\b!!"
+          # sed -i works slightly different on MacOS
+          cat ${TEMP_SCHEDULE} | sed -e "s!^test:\s*${test_name}\s*\$!!" \
```

Comment:
`sed` takes a file as an argument, cat is not needed if `${TEMP_SCHEDULE}` is single file, right? I'm surprised shellcheck doesn't complain, maybe it doesn't understand that it's one file.

Reply:
The problem is that `sed -i` [work different on MacOS](https://www.themoderncoder.com/fix-sed-i-error-macos/#:~:text=In%20other%20words%2C%20when%20using,backup%20instead%20of%20just%20%2Di%20.) and it was leading to warnings on the CI if we set `SKIPS` variable. Doing it in this way is more portable between Linux and MacOS.

Reply:
Yes, I understand you need to remove `-i` for compatibility, but probably even on MacOS `cat _file_ | sed` can be written as `sed .... _file_`? The shellcheck warning I was thinking about is this: https://www.shellcheck.net/wiki/SC2002

Anyway, it's not important, just wondering.

Reply:
Ohhh... I'm so dumb (or perhaps a bit tired). From the link I've provided above:

![image](https://github.com/timescale/timescaledb/assets/612482/9ea7de08-5ebe-4026-add5-d9430b6f73d4)

Simplified it by using `-i.backup` and it works for Linux and MacOS